XmlSchemas/ConfigurationOverrides/overrides.xsd

```diff
@@ -80,6 +173,9 @@
                 </xsd:complexType>
               </xsd:element>
               <xsd:element minOccurs="0" name="LegacyAliases">
+                <xsd:annotation>
+                  <xsd:documentation>ADVANCED CONFIGURATION. These are legacy alias for service cmdlets. If possible, use LegacyAlias on each cmdlet instead of this. This is used 1) for non-generated ("advanced") cmdlets 2) when a cmdlet requires multiple aliases.</xsd:documentation>
```

Comment:
1) for => for: 1) non-gen...

Reply:
I am adding the `:` but I am keeping the `for` after the `1)` because `2)` is followed by `when` instead.